x-pack/plugins/fleet/public/hooks/use_platform.tsx

```diff
@@ -8,7 +8,7 @@
 import { useState } from 'react';
 import { i18n } from '@kbn/i18n';
 
-export type PLATFORM_TYPE = 'linux' | 'mac' | 'windows' | 'rpm' | 'deb';
+export type PLATFORM_TYPE = 'linux' | 'mac' | 'windows' | 'rpm' | 'deb' | 'kubernetes';
 
 export const PLATFORM_OPTIONS: Array<{
```

Comment:
Here we could simply rename the current `PLATFORM_OPTIONS` to `REDUCED_PLATFORM_OPTIONS` and then export another constant. This way you could avoid searching and splicing the array in the `PlatformSelector`:
```
export const PLATFORM_OPTIONS = [
  ...REDUCED_PLATFORM_OPTIONS,
  {
    id: 'kubernetes',
    label: i18n.translate('xpack.fleet.enrollmentInstructions.platformButtons.kubernetes', {
      defaultMessage: 'Kubernetes',
    }),
    'data-test-subj': 'platformTypeKubernetes',
  },
];
```

Reply:
This is a great suggestion!